src/resources/views/crud/columns/checklist_dependency.blade.php

```diff
@@ -0,0 +1,81 @@
+@include('crud::columns.checklist', ['column' => $column['subfields']['primary']]) <br />

```

Comment:
EXCELLENT job on this one @munjaldevelopment 👏👏👏 Looks and works very well.

I have two comments here: 

1) I see there's a space before the first comma, on both rows:

![CleanShot 2022-10-21 at 06 10 49@2x](https://user-images.githubusercontent.com/1032474/197109945-b698876b-9926-41bf-a775-68519ec751fe.png)

2) It's a bit difficult to understand which is which, by looking at the column. Yes, Roles are on the first row, and Permissions are on the second, but... wouldn't know... perhaps we should add a bold "**Role:** " and "**Permission:** " at the start of each row? But then... that would be repeated on all rows, so it would become tiring... I don't know... I'm just mentioning a potential problem I spotted. Let's not do anything about it, see if Pedro spots it too.

Reply:
Added `bold label` & removed extra comma space from "role" & "permission"